lib/core.js

```diff
@@ -702,7 +702,7 @@ declare var exports: any;
 
 /* Commonly available, shared between node and dom */
 declare var console: {
-  assert(condition: boolean, ...data: Array<any>): void;
+  assert(condition: any, ...data: Array<any>): void;
```

Comment:
Sure, though I don't entirely follow the benefits of doing so. Can you give an example error that would pick up?